sdk/README.md

```diff
@@ -15,11 +15,11 @@ for [Tekton](https://github.com/tektoncd/pipeline).
   - [Tested Pipelines](#tested-pipelines)
   - [How to use the KFP-Tekton Compiler](#how-to-use-the-kfp-tekton-compiler)
     - [Installation](#installation)
-    - [Compiling a Kubeflow Pipelines DSL script](#compiling-a-kubeflow-pipelines-dsl-script)
-    - [Running the compiled pipeline on a Tekton cluster](#running-the-compiled-pipeline-on-a-tekton-cluster)
+    - [Compiling a Kubeflow Pipelines DSL Script](#compiling-a-kubeflow-pipelines-dsl-script)
+    - [Running the Pipeline on a Tekton Cluster](#running-the-pipeline-on-a-tekton-cluster)
   - [Build Tekton from Master](#build-tekton-from-master)
-  - [Additional Features](#additional-features)
-    - [Compile Kubeflow Pipelines with Artifacts Enabled](#compile-kubeflow-pipelines-with-artifacts-enabled)
+  - [Removable Features](#removable-features)
```

Comment:
"Removable" sounds very odd here. "Deprecated" maybe? Maybe "Optional" or keep "Additional" even if it means a feature being removed